package.json

```diff
@@ -17,14 +17,24 @@
   "engines": {
     "node": ">=18"
   },
+  "files": [
```

Comment:
This replaces .npmignore, right? Feel free to remove that file.

Reply:
yes, it defines the files published to npm. But TBH I thought about it, if there is an `.npmignore` and we want to make as few changes as possible the part I added in package.json is not needed and I'll delete that one in favor of `.npmignore`

Reply:
anyhow i want to mention that the test files and the lint config files should be committed along the npm package. this is not a real needs but will raise the "quality" parameter in the npmjs search results

https://www.npmjs.com/search?q=gettext